### PR TITLE
feat: json to yaml everything

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,235 +95,226 @@ To configure the package behaviour, you can modify your `network_params.json` fi
...
-        "interval_between_transactions": 1
-      }
-    },
-    // A list of locators for grafana dashboards to be loaded be the grafana service
-    "grafana_additional_dashboards": []
-}
+```yaml
+#  Specification of the participants in the network
+participants:
+  # The typoe of EL client that should be started
```
Comment (h4ck3rk3y) [inline README.md:101]:
```suggestion
  # The type of EL client that should be started
```

File: README.md [hunk 2]
```diff
@@ -95,235 +95,226 @@ To configure the package behaviour, you can modify your `network_params.json` fi
...
+  # Count of nodes to spin up for this participant
+  # Default to 1
+  count: 1
+
+  # Count of the number of validators you want to run for a given participant
+  # Default to null, which means that the number of validators will be using the network parameter num_validator_keys_per_node
+  validator_count: null
+
+#  Default configuration parameters for the Eth network
+network_params:
```
Comment (h4ck3rk3y) [inline README.md:198]: indentation

File: README.md [hunk 3]
```diff
@@ -95,235 +95,226 @@ To configure the package behaviour, you can modify your `network_params.json` fi
...
+  # Snooper can be enabled with the `snooper_enabled` flag per client or globally
+  # Defaults to false
+  snooper_enabled: false
+
+  # Count of nodes to spin up for this participant
+  # Default to 1
+  count: 1
+
+  # Count of the number of validators you want to run for a given participant
+  # Default to null, which means that the number of validators will be using the network parameter num_validator_keys_per_node
```
Comment (h4ck3rk3y) [inline README.md:194]: indentation

**Review by h4ck3rk3y — COMMENTED** [review]
Will have to change `config.yml` in circle! also lets run all tests for a bit

File: README.md (hunk 2 above)
Comment (barnabasbusa) [reply README.md:198]: whats wrong with the indentation here?

File: README.md (hunk 3 above)
Comment (barnabasbusa) [reply README.md:194]: ?

Comment (h4ck3rk3y) [reply README.md:194]: i was wondering if he `network parameter` can be on a separate line so that its justiified with the # Default

Comment (barnabasbusa) [reply README.md:198]: network parameter is a few lines below this tho. I still don't quite get what you want me to change.

Comment (h4ck3rk3y) [reply README.md:198]: the default isn't in the same vertical line as 'parameter'
Screenshot 2023-10-25 at 1 03 35 PM
```suggestion
  # Default to null, which means that the number of validators will be using the
  # network parameter num_validator_keys_per_node
```

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**